### PR TITLE
BO-83 Add tmpfiles.d config to recreate /var/run/mysqld

### PR DESCRIPTION
**Pull request by ppp0**

### CODE REVIEW

**Comment by ppp0** [comment]
See manpage for `tmpfiles.d(5)` on jessie (because of v215)

Like this, at boot time, the dir will be recreated with the correct perms. Actually, I think conf file this should go into the deb package!

File: modules/mysql/templates/tmpfiles.d/mysql.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ # (Re-)Create the temp directory at boot time
+ d /run/mysql 0755 mysql mysql
```
Comment (kris-lab) [inline modules/mysql/templates/tmpfiles.d/mysql.conf]: What is exactly the source of the problem, `/var/run/mysqld` or `/run/mysql` ?

**Comment by ppp0** [comment]
there was a typo. @kris-lab please review

**Comment by kris-lab** [comment]
Shouldn't it be ` /var/run/mysqld` like configured https://github.com/cargomedia/puppet-packages/blob/master/modules/mysql/manifests/server.pp#L30 ?

**Comment by ppp0** [comment]
@kris-lab of course it should! Thanks for reading carefully

**Comment by ppp0** [comment]
I would suggest to proceed as usual when critical pieces are affected:
* stop puppet on all clustered dbs (ads, gearman, denk not affected)
* apply manually on one of the replicas to see if service gets a restart (should not!)
* restart puppet everywhere

I tested this locally and
* There is no "daemon" to be reloaded as in `daemon-reload`
* The config files in `tmpfiles.d` are read at boot time
See
```
# systemd-analyze blame
[..]
            45ms systemd-tmpfiles-clean.service
[..]
            39ms systemd-tmpfiles-setup-dev.service
[..]
            18ms systemd-tmpfiles-setup.service
[..]
```

**Pull request merged**